spa/server.js

```diff
@@ -115,28 +115,35 @@ io.on('connection', async function (socket) {
       }
     }
   });
-  
+
   //handle shared events
   socket.on("SE", function (msg) {
     console.log(msg);
     io.to(USERS.get(socket).room).emit("SE", msg);
   });
-  
+
   //handle chat messages
   socket.on("CHAT", (chatData) => {
     console.log('chat message...');
     console.log(chatData);
     if (!chatData || !chatData.msg || typeof chatData.msg !== "string") return;
     const user = USERS.get(socket);
-    if (user?.room) {
-      webhookMessage(`${user.username} in ${user.room}`, chatData.msg);
-      io.to(user.room).emit("CHAT", {
-        username: user.username,
-        msg: chatData.msg,
-      });
+    const bannedwords = /(nigger)|(chinc)/i;
```

Comment:
Need more words to be added to list (add by putting words in parenthesis and putting vertical bar between them.)